__tests__/problem-matchers.test.js

```diff
@@ -25,13 +25,13 @@ async function testElixirMixCompileWarning() {
   ).pattern
 
   const firstOutput =
-    'warning: variable "err" does not exist and is being expanded to "err()", please use parentheses to remove the ambiguity or change the variable name'
```

Comment:
Not really important, but `npm format` should do this for you. Doesn't it?

Reply:
`npm format` won't break up a line if it's too long

Reply:
Fun stuff!